crates/matrix-sdk-ui/src/room_list_service/state.rs

```diff
@@ -260,9 +302,9 @@ macro_rules! actions {
 impl Actions {
     actions! {
         none => [],
-        first_rooms_loaded => [SetAllRoomsListToGrowingSyncMode, SetInvitesListToGrowingSyncMode, AddVisibleRoomsList],
-        recover_lists => [SetAllRoomsListToGrowingSyncMode, SetInvitesListToGrowingSyncMode],
-        reset_lists => [SetAllRoomsListToSelectiveSyncMode, SetInvitesListToSelectiveSyncMode],
+        first_rooms_are_loaded => [SetAllRoomsListToGrowingSyncMode, SetInvitesListToGrowingSyncMode, AddVisibleRoomsList],
+        first_rooms_are_loaded_again => [SetAllRoomsListToGrowingSyncMode, SetInvitesListToGrowingSyncMode, SetVisibleRoomsListToDefaultTimelineLimit],
+        prepare_to_load_first_rooms => [SetAllRoomsListToSelectiveSyncMode, SetInvitesListToSelectiveSyncMode, SetVisibleRoomsListToZeroTimelineLimit],
```

Comment:
I'm not a big fan of these names: the struct is called "Action", and there's no verb in the names here, plus they aren't very self-descriptive to me, at all. It's not a big deal, but could we name them respectively, something more obvious, like:

- `start_growing_and_add_visible`
- `finish_recovering`
- `start_recovering`

Reply:
Agreed on new names in a 1:1.